be/src/vec/exec/scan/scanner_scheduler.cpp

```diff
@@ -289,6 +297,7 @@ void ScannerScheduler::_scanner_scan(std::shared_ptr<ScannerContext> ctx,
                     scan_task->cached_blocks.back().first.get()->set_columns(
```

Comment:
vectorized::MutableBlock mutable_block(
                            scan_task->cached_blocks.back().first.get());

should - mutable block.allocated bytes before this line
and + mutable block.allocated bytes after this line.